supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -152,6 +166,23 @@
 
 *See also*:
 
+[discrete]
+[[privilege]]
+==== image:images/yes.png[yes] permission (noun)
+*Description*: A "privilege" is a right granted to an agent to perform certain restricted actions. A user is one example of an agent.
+"Privilege" generally refers to an interaction of an agent with the system or with an application rather than with a filesystem and its objects.
```

Comment:
```suggestion
"Privilege" generally refers to an interaction of an agent with the system or with an application rather than with a file system and its objects.
```